From 831b68dce042dc9078b6d33d062ece960fe64b91 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Maxime=20M=C3=A9r=C3=A9?= <maxime.mere@foss.st.com>
Date: Tue, 21 May 2024 15:22:51 +0200
Subject: [PATCH 0845/1695] dts: stm32: enable HASH on stm32mp215f-dk
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Sets HASH peripheral status to okay.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 349b13891eddc72aac3dc24043b2e8206db097bd dts: stm32: enable HASH on stm32mp215f-dk]
Signed-off-by: Maxime Méré <maxime.mere@foss.st.com>
Change-Id: Iaaeae774c9a0561f4f1920d006d1e68402c8f9cd
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/381350
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 core/arch/arm/dts/stm32mp215f-dk.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/core/arch/arm/dts/stm32mp215f-dk.dts b/core/arch/arm/dts/stm32mp215f-dk.dts
index fe66f3071..e5f6f47ad 100644
--- a/core/arch/arm/dts/stm32mp215f-dk.dts
+++ b/core/arch/arm/dts/stm32mp215f-dk.dts
@@ -49,6 +49,10 @@
 	cpu-supply = <&vddcpu>;
 };
 
+&hash1 {
+	status = "okay";
+};
+
 &rcc {
 	st,c1msrd = <2>;
 	st,clk_opp {
-- 
2.47.2

